types/node/stream.d.ts

```diff
@@ -1179,7 +1179,8 @@ declare module 'stream' {
             : (err: NodeJS.ErrnoException | null) => void;
         type PipelinePromise<S extends PipelineDestination<any, any>> = S extends PipelineDestinationPromiseFunction<any, infer P> ? Promise<P> : Promise<void>;
         interface PipelineOptions {
-            signal: AbortSignal;
+            signal?: AbortSignal;
+            end?: boolean;
```

Comment:
Looks good!

Maybe also allowing to pass `undefined` in options for users with turned on [exactOptionalPropertyTypes](exactOptionalPropertyTypes) as in another options:
```ts
            signal?: AbortSignal | undefined;
            end?: boolean | undefined;
```

Reply:
Yes you right, i updated it